app/src/main/java/com/ybsystem/tweetmate/fragments/dialog/PreviewAppearanceDialog.java

```diff
@@ -0,0 +1,39 @@
+package com.ybsystem.tweetmate.fragments.dialog;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.fragment.app.DialogFragment;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentTransaction;
+
+import com.ybsystem.tweetmate.R;
+import com.ybsystem.tweetmate.fragments.timeline.ProfileTimeline;
+import com.ybsystem.tweetmate.models.entities.Column;
+
+import static com.ybsystem.tweetmate.models.enums.ColumnType.USER_TWEET;
+
+public class PreviewAppearanceDialog extends DialogFragment {
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        // Inflate
+        View view = inflater.inflate(R.layout.dialog_preview_appearance, null);
+
+        // Create timeline
+        Fragment fragment = new ProfileTimeline().newInstance(
+                new Column(7080152L, "", USER_TWEET, false),
```

Comment:
This "7080152L" is a magic number. (Self-review)

Reply:
Fixed: https://github.com/ybs32/TweetMate-for-Android/pull/67/commits/553f65ad879d79416490cc616857fe7f7a43b652